apps/nearbyflight/nearby_flight.star

```diff
@@ -0,0 +1,196 @@
+"""
+Applet: Nearby Flight
+Summary: Show a Nearby Flight
+Description: Shows a nearby flight by querying airframes.io.
+Author: kevinelliott
+Homepage: https://github.com/airframesio & https://github.com/kevinelliott
+"""
+
+load("cache.star", "cache")
+load("encoding/json.star", "json")
+load("http.star", "http")
+load("qrcode.star", "qrcode")
+load("render.star", "render")
+load("schema.star", "schema")
+load("secret.star", "secret")
+
+AIRFRAMES_NEARBY_URL = "https://api.airframes.io/v1/tidbyt/nearby/flights"
+
+# AIRFRAMES_NEARBY_URL = "http://localhost:3001/v1/tidbyt/nearby/flights"
+AIRFRAMES_NEARBY_RADIUS = "5"
+AIRFRAMES_NEARBY_LIMIT = "5"
+AIRFRAMES_NEARBY_SORT = "distance"
+AIRFRAMES_NEARBY_SORT_ORDER = "asc"
+ENCRYPTED_API_KEY = ""
+
+def main(config):
+    api_key = secret.decrypt(ENCRYPTED_API_KEY) or config.get("api_key")
```

Comment:
Unlike in pixlet, `secret.decrypt()` failure will actually result in a hard error when running in our backend environment. Since `ENCRYPTED_API_KEY` is blank, this will always fail.